docs/how-to/deploy-cluster.md

```diff
@@ -0,0 +1,143 @@
+2. Specify in `.buildkite/pipeline.yml` and `appveyor.yml` branch of cardano-sl repository which `cardano-node` will be taken from.
+
+   This branch should be branch from the previous step and should contain added configurations.
+   Make sure that backed API compatiable with Daedalus frontend, [this page](https://github.com/input-output-hk/internal-documentation/wiki/Daedalus-installer-history) may be useful. It reflects  correspondence between cardano revisions and Daedalus revisions.
+   
+   In `appveyor.yml` in [this line](https://github.com/input-output-hk/daedalus/blob/release/0.9.0/appveyor.yml#L26)
+   you should specify your branch instead of `release/1.1.0`.
+   
+   In `.buildkite/pipeline.yml` in [this](https://github.com/input-output-hk/daedalus/blob/release/0.9.0/.buildkite/pipeline.yml#L6) and 
+   [this](https://github.com/input-output-hk/daedalus/blob/release/0.9.0/.buildkite/pipeline.yml#L14) lines
+   you need to specify your branch instead of `release/1.1.0`.
+3. Specify IP address of relay node in `installers/wallet-topology.yaml`.
+
+   To get this IP address you should execute `io info` in `nix-shell` for your cluster, 
+   IP address of node with name `r1` is needed address.
+   After you get it you should write down it in `installers/wallet-topology.yaml`, like:
+   ```
+   wallet:
+     relays: [[{"addr": "<your ip>"}]]
+     valency: 1
+     fallbacks: 3
+   ```
+   For instance, if IP is `18.196.10.154` the `wallet-topology.yaml` will look:
+   ```
+   wallet:
+     relays: [[{"addr": "18.196.10.154"}]]
+     valency: 1
+     fallbacks: 3
+   ```
+4. Update Daedalus configs.
+
+   There are two configs: `installers/launcher-config-mac.yaml` and `installers/launcher-config-windows.yaml` which should be updated.
+   First of all you need to know system start. You can take it from `config.yaml`'s `systemStart` contents.
+   
+   You need to specify `key` [here](https://github.com/input-output-hk/daedalus/blob/release/0.9.0/installers/launcher-config-mac.yaml#L42) as `devnet_staging_macos64`, 
+   and `key` [here](https://github.com/input-output-hk/daedalus/blob/release/0.9.0/installers/launcher-config-windows.yaml#L42) as `devnet_staging_win64`. 
+   To be more precisely you should specify them as your configuration names which you created in `configuration.yaml`.
+   
+   You need to specify `systemStart` [here](https://github.com/input-output-hk/daedalus/blob/release/0.9.0/installers/launcher-config-mac.yaml#L43) and [here](https://github.com/input-output-hk/daedalus/blob/release/0.9.0/installers/launcher-config-windows.yaml#L42).
+   
+   You need to specify `--update-server` [here](https://github.com/input-output-hk/daedalus/blob/release/0.9.0/installers/launcher-config-mac.yaml#L12) and [here](https://github.com/input-output-hk/daedalus/blob/release/0.9.0/installers/launcher-config-windows.yaml#L12) as `https://s3.eu-central-1.amazonaws.com/update-system-testing`.
```

Comment:
No, no, no, it's a very bad idea to refer to files by branch name. Please use concrete revisions instead. This branch may be deleted in the future or at least some changes may be done there.
Example: https://github.com/input-output-hk/daedalus/blob/66c6bca3ca1824deb77440cb4e3cb753ba0d9e50/installers/launcher-config-windows.yaml#L12

Note: you can press `y`.

Reply:
if the branch is deleted, this commit also will be unavailable, won't it?

Reply:
Commit may belong to more than one branch. If at least one branch with this commit is not deleted, the commit itself won't be deleted. So just choose commits from stable branches (e. g. `develop`).